devito/core/cpu.py

```diff
@@ -90,6 +90,8 @@ def _normalize_kwargs(cls, **kwargs):
         o['blockinner'] = oo.pop('blockinner', False)
         o['blocklevels'] = oo.pop('blocklevels', cls.BLOCK_LEVELS)
 
+        o['skewing'] = oo.pop('skewing', False)
```

Comment:
Each though it will be a different pass, I _think_ you can put this option right below "blocklevels" (so no blank line) because essentially "skewing" is made for "blocking" (and nothing else, right?)